Runtime/AvatarCreator/Scripts/UI/Elements/AvatarListElement.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ReadyPlayerMe.Core;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.Serialization;
+using UnityEngine.UIElements;
```

Comment:
Keep an eye out on the unused using statements in Rider (greyed out lines) as they can all be removed 
![image](https://github.com/readyplayerme/rpm-unity-sdk-core/assets/7085672/04dea7e8-1cda-424c-8bdd-a43919c69ae4)